quarkus/runtime/src/main/java/org/keycloak/provider/quarkus/QuarkusLifecycleObserver.java

```diff
@@ -38,10 +37,7 @@
     private static final String KEYCLOAK_ADMIN_ENV_VAR = "KEYCLOAK_ADMIN";
     private static final String KEYCLOAK_ADMIN_PASSWORD_ENV_VAR = "KEYCLOAK_ADMIN_PASSWORD";
 
-    @Inject
-    KeycloakApplication application;
-
-    void onStartupEvent(@Observes StartupEvent event) {
+    private void onStartupEvent(@Observes StartupEvent event) {
```

Comment:
If you use private you are forcing reflection. That is why the method is marked with default/package.